hdf5-types/src/h5type.rs

```diff
@@ -40,18 +40,24 @@ impl IntSize {
 
 #[derive(Clone, Copy, Debug, PartialEq, Eq, PartialOrd, Ord)]
```

Comment:
Same here with `#[cfg_attr(not(feature = "f16"), non_exhaustive)]`

Reply:
Tried it out, but how would you do this then: https://github.com/aldanor/hdf5-rust/blob/46f2bccc30e1f440484e9fcf3bb5e35e6804de05/hdf5/src/hl/datatype.rs#L354-L359

Reply:
Yeah, it seems bad having to panic there. Very fringe case so skip my suggestion